src/main/java/org/tdl/vireo/model/repo/impl/UserRepoImpl.java

```diff
@@ -30,8 +48,21 @@ public User create(String email, String firstName, String lastName, AppRole role
 
         newUser.putSetting("id", newUser.getId().toString());
         newUser.putSetting("displayName", newUser.getFirstName() + " " + newUser.getLastName());
-        newUser.putSetting("preferedEmail", newUser.getEmail());
+        newUser.putSetting("preferedEmail", newUser.getEmail());        
         newUser.setActiveFilter(activeFilter);
+        
+        List<SubmissionListColumn> defaultFilterColumns = null;
+		try {
```

Comment:
We now have a nice in memory default settings service. It should be used for this. I don't think the initial defaults are meant to be real time modified. Not sure user creation should have file I/O operations.